app/controllers/miq_policy_controller.rb

```diff
@@ -322,6 +322,18 @@ def fetch_log
                        :message => _("Policy log downloaded"))
   end
 
+  # Not necessarily a whole id, but the kind of lowerCamelCase we use for model names
+  # within ids like 'po_xx-control_xx-control-containerImage'.
+  def ui_model_from_id(id)
+    return nil if id.nil?  # TODO: fix callers, remove
```

Comment:
Why the `nil` here? How would this change affect other routines? (esp. `get_json`)